plugins/option.go

```diff
@@ -77,25 +76,13 @@ func MetricsHost(h string) Option {
 func MetricsPort(port int) Option {
 	return func(p *Plugin) error {
 		if port <= 65535 {
-			p.MetricsPort = strconv.Itoa(port)
+			p.MetricsPort = port
```

Comment:
i'd probably leave it an int and call `strconv.Itoa` rather then have a port as a string